doc/pins/p4rt_app_hld.md

```diff
@@ -164,6 +165,30 @@ P4RT application introduces new tables that are written to APPL_DB for the table
 
 The P4RT application code will have unit & component tests that together will give >80% code coverage.
 
+## Configuring P4RT Application
+
+The P4RT application is configured at the start-up by reading the P4RT configuration from the CONFIG_DB. If no valid config exists in CONFIG_DB, it uses the default values. The configuration can be added to the CONFIG_DB either manually (by adding to config_db.json) or through gNMI, as it can be different for users.
```

Comment:
I don't think most of these can be configured over gNMI. I would suggest leaving that part out.